emission/net/usercache/formatters/android/survey.py

```diff
@@ -9,16 +9,18 @@ def format(entry):
     formatted_entry = ad.AttrDict()
     formatted_entry["_id"] = entry["_id"]
     formatted_entry.user_id = entry.user_id
-    
+
     metadata = entry.metadata
     if "time_zone" not in metadata:
         metadata.time_zone = "America/Los_Angeles"
-    logging.debug("Timestamp conversion: %s -> %s done" % (entry.metadata.write_ts, metadata.write_ts))
-    fc.expand_metadata_times(metadata)
+    if "write_ts" in metadata:
```

Comment:
I am not sure this is going to work - your error was with the `ts` field, not the `write_ts` field.
Did you try re-running to see whether the error is resolved?
https://github.com/e-mission/e-mission-server/wiki/Troubleshooting-tips-(FAQ)#how-do-i-recover